roles/ruby_s/molecule/default/converge.yml

```
@@ -0,0 +1,15 @@
+---
+- name: Converge
+  hosts: all
+  vars:
+    - running_on_server: false
+  become: true
+  pre_tasks:
+    - name: update cache
+      apt:
+        update_cache: true
+        cache_valid_time: 600
+  tasks:
+    - name: "Include ruby"
+      include_role:
+        name: ruby
```

We want this to run the `ruby_s` role, not the `ruby` role.
```suggestion
        name: ruby_s
```